notes/RELEASING.md

```diff
@@ -0,0 +1,23 @@
+## Releasing
```

Comment:
I think I like having this file in the root of the project, easier to find.

Reply:
Yep, good point, moved back.